compiler/rustc_error_codes/src/error_codes/E0705.md

```diff
@@ -1,3 +1,5 @@
+#### Note: this error code is no longer emitted by the compiler.
```

Comment:
You'll need to remove `E0705` from the code blocks labels below as well.

Reply:
OK, I was a little confused why that wasn't generating an error. The `rust2018` tag was incorrect, and was causing rustdoc to ignore the code block.

Reply:
Oh interesting. Might be interesting to add a check for that then. Adding that to my todo list.